### PR TITLE
Zach Campbell

### PR DESCRIPTION
**Pull request by wajnurfes**

### CODE REVIEW

File: friends/friendController.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    }
+    return obj;
+  }
+
+  return (req, res, next) => {
+    const { firstName, lastName, age } = req.body;
+    req.saneBody = clean({ firstName, lastName, age });
+    next();
+  }
+})();
```
Comment (nphillips78) [inline friends/friendController.js:21]: I love your 'saneBody' - good work

File: friends/friendModel.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  lastName: {
+    type: String,
+    required: true
+  },
+  age: {
+    type: Number,
+    min: 1,
+    max: 120,
+    required: true
+  },
```
Comment (nphillips78) [inline friends/friendModel.js:17]: nested schema for the win

**Review by nphillips78 — COMMENTED** [review]
Good job, as usual. I do have a question - am I just overlooking your validation? I don't see any error handling for missing name, age, etc?

**Comment by wajnurfes** [comment]
Hm... looks like I left that off.  I had done it in the sanitize middleware before, I think.  I just forgot to reimplement it for this project when I adapted it from last project.

**Pull request closed without merging**